src/Chapter14/Listing14.09.OnTemperatureChangedThrowingAnException.cs

```diff
@@ -13,27 +12,28 @@ public static void Main()
             Heater heater = new Heater(60);
             Cooler cooler = new Cooler(80);
 
-            // Using C# 2.0 or later syntax
             thermostat.OnTemperatureChange +=
                 heater.OnTemperatureChanged;
-            // Using C# 3.0.  Change to anonymous method
-            // if using C# 2.0
+            #region HIGHLIGHT
             thermostat.OnTemperatureChange +=
                 (newTemperature) =>
                     {
                         throw new InvalidOperationException();
                     };
+            #endregion HIGHLIGHT
             thermostat.OnTemperatureChange +=
                 cooler.OnTemperatureChanged;
 
             Console.Write("Enter temperature: ");
             string? temperature = Console.ReadLine();
             if (!int.TryParse(temperature, out int currentTemperature))
             {
-                Console.WriteLine($"'{temperature}' is not a valid integer.");
+                Console.WriteLine
+                    ($"'{temperature}' is not a valid integer.");
```

Comment:
I think splitting between two lines at this exact point in time may be confusing. Maybe something more similar to this if we are trying to shorten the lines?
```
                Console.WriteLine($"'{temperature}' 
                is not a valid integer.");
```
But I think maybe one line could work well here, thoughts?

Reply:
I changed it to be like you said, it adds an extra "$" so just make sure it's what you want!